CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased]
 
+* Removing support for Python 3.8
```

Comment:
Why is this needed? pydantic 2 does support python 3.8 https://github.com/pydantic/pydantic/blob/main/pyproject.toml#L65